Gemfile

```diff
@@ -11,6 +11,7 @@ gem "ohai", "~> 13"
 
 gem "chef-config", path: File.expand_path("../chef-config", __FILE__) if File.exist?(File.expand_path("../chef-config", __FILE__))
 gem "cheffish", "~> 13" # required for rspec tests
+gem "american_date"
```

Comment:
This seems like a gem we probably shouldn't be bringing in. It's about 50 lines of code and adds yet another dependency to Chef. Instead of adding this dependency can we just add the same logic to convert date stamps to the "american" style date stamp?